moveit_py/README.md

```diff
@@ -0,0 +1,28 @@
+# MoveIt 2 Python Library
+<img src="./banner.png" width="50%">
+
+`moveit_py` is a Python library for interfacing with the core functionalities of Moveit 2.
+The goal of this library is to provide a simplified interface for Moveit 2 Python users.
```

Comment:
Correct to "Moveit 2" to "MoveIt 2" (capital "i") in both places

Reply:
https://github.com/ros-planning/moveit2/pull/1546/commits/9ee2566d45844b6317cbacabb59ddbc9bd3d70e8